README.md

```diff
@@ -93,5 +94,14 @@ You can define this fallback url parameter like you define ids. The difference i
 </script>
 ```
 
+### load a module
+to load javascript modules you have to add the suffix `.module` to the url, this will load the script with the `type` set on `module`
```

Comment:
This is just a matter of preferences but i prefer a prefix "module:" instead of a ".module" suffix